Makefile

```diff
@@ -150,8 +167,18 @@ docker-create-push-manifests:	## Create and push manifests for a docker image (d
 			docker manifest create $(MANIFEST_IMAGE_NAME):latest --amend $(MANIFEST_IMAGE_NAME):latest-amd64 --amend $(MANIFEST_IMAGE_NAME):latest-arm64 && \
 		((! (git diff HEAD~1 localstack/__init__.py | grep '^+__version__ =') && \
 				echo "Only pushing tag 'latest' as version has not changed.") || \
-			(docker manifest create $(MANIFEST_IMAGE_NAME):$(IMAGE_TAG) --amend $(MANIFEST_IMAGE_NAME):$(IMAGE_TAG)-amd64 --amend $(MANIFEST_IMAGE_NAME):$(IMAGE_TAG)-arm64 && \
-				docker manifest push $(MANIFEST_IMAGE_NAME):$(IMAGE_TAG))) && \
+			(docker manifest create $(MANIFEST_IMAGE_NAME):$(IMAGE_TAG) \
+			--amend $(MANIFEST_IMAGE_NAME):$(IMAGE_TAG)-amd64 \
+			--amend $(MANIFEST_IMAGE_NAME):$(IMAGE_TAG)-arm64 && \
+			docker manifest create $(MANIFEST_IMAGE_NAME):$(MAJOR_VERSION).$(MINOR_VERSION) \
+			--amend $(MANIFEST_IMAGE_NAME):$(MAJOR_VERSION).$(MINOR_VERSION)-amd64 \
+			--amend $(MANIFEST_IMAGE_NAME):$(MAJOR_VERSION).$(MINOR_VERSION)-arm64 && \
+			docker manifest create $(MANIFEST_IMAGE_NAME):$MAJOR_VERSION.$(MINOR_VERSION).$(PATCH_VERSION)
```

Comment:
You forgot the brackets here for `MAJOR_VERSION`.

Reply:
Omg, I gotta get some sleep.